lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -275,11 +251,11 @@ spec = do
                 ]
 
     Hspec.it "SHELLEY_MIGRATE_02 - \
-        \migrating wallet with dust should fail."
+        \migrating wallet with 'dust' (that complies with minUTxOValue) should pass."
         $ \ctx -> do
             -- NOTE
             -- Special mnemonic for which wallet has dust
-            -- (10 utxo with 33 lovelace)
+            -- (10 utxo with 43 ADA)
```

Comment:
Not so dusty anymore :sweat_smile: ... Does this test still has meaning? (we do test similar wallet migrations from other scenarios, don't we?)

Reply:
> Not so dusty anymore

Indeed, however It's still sort of "dust" that complies with minUTxOValue I guess...

There are two of those tests now:
`SHELLEY_MIGRATE_02 - migrating wallet with 'dust' (that complies with minUTxOValue) should pass.` (this one)
`SHELLEY_MIGRATE_01_big_wallet`

where in both "dust" is now 1 ada... in the end I'd leave them both as they are still a bit different than just `fixtureWallet`...